.travis.yml

```diff
@@ -1,17 +1,34 @@
 language: node_js
 sudo: false
-node_js:
-- "8"
-- "10"
-- "12"
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    packages:
-    - gcc-4.9
-    - g++-4.9
-before_install:
-- export CC="gcc-4.9" CXX="g++-4.9"
+node_js: 10
+matrix:
+  include:
+  - os: linux
+    env: MATRIX_EVAL="CC=gcc-4.9 && CXX=g++-4.9"
```

Comment:
What is `MATRIX_EVAL`?

Reply:
I modified the .travis file to specify the environment variables explicitly.  The MATRIX_EVAL is just one way to setup the environment when dealing with a matrix build, which is now necessary to prebuild for both OSX and Linux.